### DISCUSSION TITLE
Add sorts to various random package views

### DISCUSSION THREAD

**Discussion by bnb** [community/community#128403] [npm]
Something that's always been super frustrating to me is that there's a handful of views on npmjs.com that have no possible way to actually sort _massive_ lists of packages, severely limiting their utility. 

For example, when I'm looking at [querystring](http://npm.im/querystring) I can see that there's 2966 dependents that account for 12m downloads weekly. The view on the package page seems entirely random, but then the [specific `dependents` view](https://www.npmjs.com/browse/depended/querystring) is sorted differently - it does include some popular modules at the top, but I can't tell how it's actually sorted.

The same is true for personal profiles / org profiles. If you look at [my profile](https://www.npmjs.com/~bnb), the order of modules is seemingly completely random - it's not downloads, it's not most recent publish, it's not alphabetical. It's effectively useless.

I'd really like to see some basic sorts available in these UIs (and any other that similarly shows a list of packages):
- Downloads (High to Low)
- Downloads (Low to High)
- Most Recent Publish
- Least Recent Publish
- Most dependents
- Least dependents

**Comment by MylesBorins** [comment]
Hey @bnb 

We are talking about doing some refreshes to the landing page and I'll include this in the last of possible improvements

**Reply by bnb** [reply]
Very glad to hear that ❤️

**Reply by jasonwilliams** [reply]
Hey @MylesBorins is there an update to this? I agree with @bnb it can be very frustrating trying to find the most downloaded dependent of a package and going through over 300 packages individually is very tedious.

**Reply by MaTiAtSIE** [reply]
Strongly agree with this!

**Reply by GulgDev** [reply]
+1, would be very nice to have this feature
